google_auth_oauthlib/flow.py

```diff
@@ -463,6 +464,8 @@ def run_local_server(
         Args:
             host (str): The hostname for the local redirect server. This will
                 be served over http, not https.
+            bind_addr (str): The ip address for the redirect server to listen on
```

Comment:
Please document this as optional, and explain the fallback behavior if this is `None`.

Thanks!